packages/blade/src/components/StepGroup/StepLine.tsx

```diff
@@ -0,0 +1,356 @@
+import type { StepItemProps } from './types';
+import { StepItemIndicator } from './StepItemMarker';
+import { useStepGroup } from './StepGroupContext';
+import { getLineSpacings } from './tokens';
+import { Box } from '~components/Box';
+import type { BaseBoxProps } from '~components/Box/BaseBox';
+import BaseBox from '~components/Box/BaseBox';
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import Svg from '~components/Icons/_Svg';
+import { makeSize, useTheme } from '~utils';
+
+type StepLineSvgProps = {
+  isDotted?: boolean;
+} & StyledPropsBlade;
+
+const useDottedLineStyles = ({ isHorizontal }: { isHorizontal?: boolean } = {}): BaseBoxProps => {
+  const { theme } = useTheme();
+
+  const svgString = `<svg width="2" height="2" viewBox="0 0 2 2" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M0 1C0 0.447715 0.447715 0 1 0C1.55228 0 2 0.447715 2 1C2 1.55228 1.55228 2 1 2C0.447715 2 0 1.55228 0 1Z"
+      fill="${theme.colors.surface.border.gray.subtle}"
+    />
+  </svg>
+  `;
+
+  const backgroundURL = `url(data:image/svg+xml;base64,${window.btoa(svgString)})`;
```

Comment:
This might fail on SSR. Direct window access.

Reply:
yup fixed it.